### PR TITLE
Update copyright footer

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

**Comment by codificat** [comment]
Apologies for the delay here :sweat: 

Prow complains about missing DCO, but it seems to be there... let's
/retest

File: _includes/footer.html [hunk 1]
```diff
@@ -2,7 +2,11 @@
   <div class="row justify-content-center">
     <div class="col-sm-12 col-md-5">
       <p class="privacy-statement text-sm-left" style="text-align: center;">
-        &copy; <script type="text/javascript"> document.write(new Date().getFullYear()); </script> KubeVirt | <a href="/privacy" class="privacy-statement-link">Privacy Statement</a>
+        &copy; KubeVirt Authors {{ 'now' | date: "%Y" }} | Documentation Distributed under <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY-4.0</a>. | <a href="/privacy" class="privacy-statement-link">Privacy Statement</a>
```
Comment (codificat) [inline _includes/footer.html:5]: The KubeVirt project uses the Apache License, and this includes the user guide as per the [LICENSE file in its repo](https://github.com/kubevirt/user-guide/blob/master/LICENSE).

As each repo in the org already contains a LICENSE, I think we can remove this license reference from here and just leave the copyright note. Makes sense?

**Comment by lucperkins** [comment]
As I'm no longer with CNCF, I'm going to close this.

@celestehorgan Could you follow up on this?

**Comment by celestehorgan** [comment]
Will do, thanks Luc!

**Pull request closed without merging**